### PR TITLE
Fixed breaking constraint in Event Details page

### PR DESCRIPTION
**Pull request by jasonskycore**

### CODE REVIEW

File: Sounds Like A Plan/Features/Event Details V2/Card Views/AboutCardView.swift [hunk 1]
```diff
@@ -17,21 +17,28 @@ class AboutCardView: SLAPCardView {
...
         configure()
         layoutConstraints()
+        debug()
     }
 
     required init(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
     
+    private func debug() {
```
Comment (gremlinflat) [inline Sounds Like A Plan/Features/Event Details V2/Card Views/AboutCardView.swift:27]: what is this??

File: Sounds Like A Plan/Features/Event Details V2/Card Views/WhatsappCardView.swift [hunk 2]
```diff
@@ -29,23 +30,38 @@ class WhatsappCardView: SLAPCardView {
...
         super.init(frame: frame)
         configure()
+        debug()
     }
     
     required init(coder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
     
+    private func debug() {
```
Comment (gremlinflat) [inline Sounds Like A Plan/Features/Event Details V2/Card Views/WhatsappCardView.swift:40]: what is this?

**Review by gremlinflat — CHANGES_REQUESTED** [review]
minor

File: Sounds Like A Plan/Features/Event Details V2/Card Views/AboutCardView.swift (hunk 1 above)
Comment (jasonskycore) [reply Sounds Like A Plan/Features/Event Details V2/Card Views/AboutCardView.swift:27]: buat debug constraint, gw mikirnya drpd ngotorin init/didload mending gw bikin func sendiri

File: Sounds Like A Plan/Features/Event Details V2/Card Views/WhatsappCardView.swift (hunk 2 above)
Comment (jasonskycore) [reply Sounds Like A Plan/Features/Event Details V2/Card Views/WhatsappCardView.swift:40]: same

**Pull request merged**